homeassistant/components/wink.py

```diff
@@ -112,6 +112,7 @@ class WinkDevice(Entity):
 
     def __init__(self, wink, hass):
         """Initialize the Wink device."""
+        self.hass = hass
```

Comment:
I don't think that is happening correctly. Without that line I get the following error.

```python
Traceback (most recent call last):
  File "/usr/lib/python3.4/threading.py", line 911, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.4/threading.py", line 859, in run
    self._target(*self._args, **self._kwargs)
  File "/home/w1ll1am/.virtualenvs/hass_dev/config/deps/pubnub/workers.py", line 25, in run
    self._take_message()
  File "/home/w1ll1am/.virtualenvs/hass_dev/config/deps/pubnub/pubnub.py", line 341, in _take_message
    self._process_incoming_payload(msg)
  File "/home/w1ll1am/.virtualenvs/hass_dev/config/deps/pubnub/workers.py", line 85, in _process_incoming_payload
    self._listener_manager.announce_message(pn_message_result)
  File "/home/w1ll1am/.virtualenvs/hass_dev/config/deps/pubnub/managers.py", line 202, in announce_message
    callback.message(self._pubnub, message)
  File "/home/w1ll1am/.virtualenvs/hass_dev/config/deps/pubnubsubhandler.py", line 186, in message
    func(json.loads(json_data))
  File "/home/w1ll1am/.virtualenvs/hass_dev/lib/python3.4/site-packages/homeassistant-0.37.0.dev0-py3.4.egg/homeassistant/components/wink.py", line 133, in _pubnub_update
    self.update_ha_state(True)
  File "/home/w1ll1am/.virtualenvs/hass_dev/lib/python3.4/site-packages/homeassistant-0.37.0.dev0-py3.4.egg/homeassistant/helpers/entity.py", line 191, in update_ha_state
    self.async_update_ha_state(force_refresh), self.hass.loop
AttributeError: 'NoneType' object has no attribute 'loop'
```